mmv1/third_party/terraform/services/container/node_config.go.erb

```diff
@@ -675,6 +675,12 @@ func schemaNodeConfig() *schema.Schema {
 						},
 					},
 				},
+				"resource_manager_tags": {
+					Type:        schema.TypeMap,
+					Optional:    true,
+					ForceNew:    true,
+					Description: `A map of resource manager tags. Resource manager tag keys and values have the same definition as resource manager tags. Keys must be in the format tagKeys/{tag_key_id}, and values are in the format tagValues/456. The field is ignored (both PUT & PATCH) when empty.`,
```

Comment:
can this be made a bit more user friendly? eg, the command line tool allows the following formats:
- `tagKeys/123456789=tagValues/987654321`
- `12345678901/env=dev` (org number)
- `example-project/env=dev`
- `11223344556/env=dev` (project number)

([docs](https://cloud.google.com/kubernetes-engine/docs/how-to/tags-firewall-policies#tag-syntax-cli))

it appears the API accepts tags in some of those formats?

https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1/ResourceManagerTags

the change to the documentation also includes those formats https://github.com/GoogleCloudPlatform/magic-modules/pull/9531/files#diff-77af46e13814452e05453e79bc81a3fde5b5299a0f0cbb7c12f16692b4258358R908

Reply:
Hello @kamalmarhubi, I guess the schema description (taken from API discovery) is kinda missleading, but this enhancement also accepts other formats as it is mentioned in the documention.